caf-audit/src/test/java/com/hpe/caf/auditing/elastic/ElasticAuditIT.java

```diff
@@ -503,13 +506,17 @@ private static void verifyFixedFieldResult(SearchHit[] results, String field, Ob
         }
     }
 
-    private static void verifyCustomFieldResult(SearchHit[] results, String field, Object expectedValue, String type)
+    private static void verifyCustomFieldResult(SearchHit[] results, String field, Object expectedValue, String type, AuditIndexingHint indexingHint)
         throws ParseException
     {
         //  Determine entry key to look for based on type supplied.
         switch (type.toLowerCase()) {
             case "string":
-                field = field + ElasticAuditConstants.CustomFieldSuffix.KEYWORD_SUFFIX;
+                if (indexingHint == AuditIndexingHint.KEYWORD) {
```

Comment:
Not in this case since we are dealing with string type fields. We always pass an indexing hint when verifying search results for fields of this type.